sdk/monitor/azure-monitor-opentelemetry-autoconfigure/src/main/java/com/azure/monitor/opentelemetry/autoconfigure/implementation/quickpulse/QuickPulseDataCollector.java

```diff
@@ -225,7 +249,8 @@ private void addException(TelemetryExceptionData exceptionData, int itemCount,
         ExceptionDataColumns columns = new ExceptionDataColumns(exceptionData);
         applyMetricFilters(columns, "Exception", currentConfig);
 
-        if (matchesDocumentFilters(columns, "Exception", currentConfig)) {
+        List<String> documentStreamIds = new ArrayList<>();
+        if (matchesDocumentFilters(columns, "Exception", currentConfig, documentStreamIds)) {
```

Comment:
The second type could be an enum, or perhaps `TelemetryType`. In this way, the method can't accept any string for second argument. It's less error-prone.